### PR TITLE
Resources view

### LINKED ISSUE #323: Extra resources screen
As a CE100 user, I can access extra practical resources, which can not naturally be classified by a topic, in a separate place on the app, so that I have quick access to more practical documents for my day to day work.

These extra resources can be accessed from the top burger nav. An admin will determine which resources and up there by using a checkbox in the resource creation/edit view.

screen shot 2017-01-06 at 10 14 32

[link](https://zpl.io/A3NI9)
[location in nav](https://zpl.io/jcEYg)

### PR DESCRIPTION
**Pull request by mk4111**
see #323 

Adds resources view, adds tests for permissions, adding new resource and editting existing resource.

Relies on https://github.com/postgres-plugin/insights/pull/18 to be merged in first

### CODE REVIEW

File: templates/partials/resources.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="card card--challenge">
+  {{#if permissions.admin}}
+    <a class="icon icon--edit" href="/insights/{{id}}/edit">
+      <img src="../../../public/assets/Edit.svg" alt="Edit" />
+    </a>
+  {{/if}}
+  <a href={{url}} class="card__header card__header--challenge card__data--link">{{title}}</a>
+  <p class="card__copy card__additional">By {{author}}</p>
+  <p class="card__copy card__additional card__additional--small">{{type}}</p>
+</div>
```
Comment (JMurphyWeb) [inline templates/partials/resources.html]: Change for insights partial

File: test/insights/admin-adds-resource.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+// access /resources as admin, primary and secondary
+tape('/resources accessible by all users', function (t) {
+  sessions.addAll(function () {
+    init(config, function (error, server, pool) {
+      t.ok(!error, 'no initialising error');
+      server.inject(getAll('admin_1'), function (res) {
+        t.equal(res.statusCode, 200, 'admin can access /resources endpoint');
+        server.inject(getAll('primary_3'), function (res) {
+          t.equal(res.statusCode, 200, 'primary can access /resources endpoint');
+          server.inject(getAll('secondary_12'), function (res) {
```
Comment (JMurphyWeb) [inline test/insights/admin-adds-resource.test.js]: We can add `/resources` to the `/test/server/secondary-permissions` test and remove it from here.

**Comment by JMurphyWeb** [comment]
LG just a couple small changes 🌮

File: test/insights/admin-adds-resource.test.js (hunk 2 above)
Comment (mk4111) [inline test/insights/admin-adds-resource.test.js]: No I completely agree - think that's a guuurd idea :+1:

File: templates/partials/resources.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="card card--challenge">
```
Comment (SimonLab) [inline templates/partials/resources.html]: Now that the insight partial is used in the resource view could we delete this file?

Comment (mk4111) [inline templates/partials/resources.html]: Yup! Sorry my bad!

**Pull request merged**